### DISCUSSION TITLE
jkj_03_valid에 대해서

### DISCUSSION THREAD

**Discussion by JiyouSeo** [boostcampaitech2/object-detection-level2-cv-06#91] [ideas]
image_id별로 겹침 여부,겹친 area를 파악하는 코드를 작성해보려고 하는데, 혹시 괜찮을 아이디어일지 먼저 상의해보고 싶습니다.

**Comment by jaehwan-AI** [comment]
할 수 있는 EDA는 해보는 것이 좋은 것 같습니다.
무의미한 결과가 나올 수도 있지만 그래도 안해보는 것보다는 해보고 결과를 분석해보는 것이 좋을 것 같다는 생각입니다.
그런 과정에서 예상치 못한 인사이트가 도출될 수 있다고 생각합니다.

**Reply by JiyouSeo** [reply]
네, 한번 진행해서 피어세션때 공유해보도록 하겠습니다~

**Reply by kwakjeeyoon** [reply]
클래스 별로 겹친 영역이 기준 이상이면 삭제하는 등 유용하게 사용될 수 있을것 같습니다!
